README.md

```diff
@@ -7,8 +7,17 @@ This is the RAS Collection Instrument micro-service, responsible for the uploadi
 
 The API is specified [here](./API.md).
 
+## Environment
+This requires pipenv and tox to be installed:
+```
+pip install pipenv
+```
 
 ## Tests
+Tests require tox to be installed.
+```
+pip install tox
```

Comment:
Tox is already listed in the Pipfile's dev dependencies. So perhaps line 17-19 are not needed here and line 21-22 can be updated to something like:

"To run the tests, install the dev dependencies using Pipenv and run tox within the virtual environment, which will run the unit tests with py.test, then run flake8 coverage.
```
pipenv install --dev
pipenv run tox
```
"

Reply:
Good call. The confusion arose on my part because I ran the tests before pip install. Thanks for the clarification